### PR TITLE
Add mockchain info to README

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,23 @@ You'll also need an alias `kojikat` to point to:
    targeted platforms and required SCL + non-SCL modes.
 1. Submit a pull request against `rpm/develop`
 
+## HOWTO: build multiple packages
+
+If you have multiple packages that you want to build together because they
+depend on each other, `mockchain` can be very helpful. To do this:
+
+1. Use tito to generate a SRPM for every package. Run  `tito build --srpm
+--test --builder tito.builder.MockBuilder --arg mock_config_dir=mock/ --arg
```
Comment (domcleal) [inline README.md]: I don't think any of the --builder or mock arguments are required, using `tito build --srpm --test` is sufficient.

File: README.md [hunk 2]
```diff
@@ -143,6 +143,23 @@ You'll also need an alias `kojikat` to point to:
...
+mock=el7-scl` or similar (depending on your desired mock config) on every
+package directory
+1. Run `mockchain -r el7-scl --tmp_prefix tfm --recurse /tmp/tito/*.src.rpm`
+to build the SRPMs created in the previous step. This will attempt to build
+all your packages in alphabetical order and will retry.
+1. The results of mockchain will be put in '/var/tmp/mock-chain-tfm.../results/el7-scl'
+or similar. To benefit from that and avoid building packages that were
+already successfully built, run  `mockchain -l /var/tmp/mock-chain-tfm.../results/el7-scl`
```
Comment (domcleal) [inline README.md]: Usually just `-l /var/tmp/mock-chain-tfm...` without the results suffix.

**Comment by dLobatog** [comment]
Updated according to comments

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**